toolchain/lex/testdata/multifile.carbon

```diff
@@ -18,9 +18,12 @@ a;
 // CHECK:STDOUT: - filename: b.carbon
 // CHECK:STDOUT:   tokens: [
 // CHECK:STDOUT:     { index: 0, kind: 'StartOfFile', line: {{ *\d+}}, column:  1, indent: 1, spelling: '', has_trailing_space: true },
-b;
-// CHECK:STDOUT:     { index: 1, kind:  'Identifier', line: {{ *}}[[@LINE-1]], column:  1, indent: 1, spelling: 'b', identifier: 0 },
+a;
+// CHECK:STDOUT:     { index: 1, kind:  'Identifier', line: {{ *}}[[@LINE-1]], column:  1, indent: 1, spelling: 'a', identifier: 0 },
```

Comment:
We'd get this same output regardless of whether we reuse the value store across files. Maybe it'd be better to test `b;` and then `a;`?

Reply:
Done